app/api/user.py

```diff
@@ -76,7 +76,7 @@ def add_user(data):
             "'application_id' is only valid for users with applicant' or 'supplier' role, not '{}'".format(user.role))
 
     if user.role == 'buyer':
-        agency = agency_service.get_or_add_agency(get_email_domain(email_address))
+        agency = agency_service.get_or_add_agency(get_email_domain(email_address.lower()))
```

Comment:
Wondering if it might be better to `.lower()` inside `get_or_add_agency()`? in the event we use this somewhere else and forget to lower it before hand.

Reply:
updated